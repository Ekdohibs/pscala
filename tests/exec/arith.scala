
object Main {
  def main(args: Array[String]) {
    print(1 + 2 * 3); print("\n");
    print(- 2 * 3); print("\n");
    print(2 * 3 + 4 * 7); print("\n");
    print(7 / 1 / 2); print("\n");
    print(2 * - 3 * - 4); print("\n")
  }
}
