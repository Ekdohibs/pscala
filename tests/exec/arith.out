7
-6
34
3
24
